### ISSUE TITLE
.NET Core 3.1 : Framework Reference Resolution problems

### ISSUE THREAD

**Issue by philstopford** [picoe/Eto#1548]
## Expected Behavior

Code builds without issue

## Actual Behavior

Code fails to build with errors : 

```
  Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(283, 5): [NETSDK1073] The FrameworkReference 'Microsoft.WindowsDesktop.App.WindowsForms' was not recognized
  Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(283, 5): [NETSDK1073] The FrameworkReference 'Microsoft.WindowsDesktop.App' was not recognized
```

## Steps to Reproduce the Problem

This derives from trying to workaround issues #1546 and #1547 by manually editing the project files.

  1. Changing the TargetFramework in the two project files (one for the library, one for the executable) to dotnetcoreapp3.1 yields a fail to compile with the mentioned errors.
  2. Changing Project SDK to 'Microsoft.NET.Sdk.WindowsDesktop' makes those errors go away, but a new one appears (this is building with Rider under Linux Mint with dotnet 3.1 installed) : 
```
  The imported project "/usr/share/dotnet/sdk/3.1.100/Sdks/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.WinFX.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  /usr/share/dotnet/sdk/3.1.100/Sdks/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props
```
Up to now, I think Windows doesn't suffer from this problem.

## Code that Demonstrates the Problem

I've attached a ZIP file corresponding to step (2) above.

## Specifications

  - Version: 2.5.0-ci-10403
  - Platform(s): Non-Windows
  - Operating System(s): Non-Windows
  
[nc31_modified.zip](https://github.com/picoe/Eto/files/4022045/nc31_modified.zip)

**Comment by cwensley** [comment]
Thanks for the reporting the issues, @philstopford!  As I've mentioned in the other issues, targeting .NET Core will not be _default_, but should be added as an option.  It might be that for us to use the new Microsoft.NET.Sdk.WindowsDesktop sdk, we will _have_ to use separate launcher projects. It would be a shame as using a single launcher has been very convenient. 

Basically it appears the problem boils down to that you cannot compile anything using Microsoft.NET.Sdk.WindowsDesktop on anything other than Windows..

**Comment by cwensley** [comment]
This should be fixed with #1579 

It appears we cannot use a combined launcher when using .NET Core at this point due to not being able to build WinForms or WPF apps on anything other than Windows.
